src/swift/Queue.swift

```diff
@@ -12,6 +12,8 @@
 
 // dispatch/queue.h
 
+import CDispatch
```

Comment:
At first glance, I think no, but I'm not 100% positive.  This code below does fail to compile if I only import Dispatch, which suggests it isn't  leaking through

```
import Dispatch
let x:dispatch_queue_t? = nil
let g2 = dispatch_group_create()
```
```
dgrove@dgrove-vb-swift:~/swift/tests$ swiftc -Xcc -fblocks -Xcc -D__DISPATCH_BUILDING_SWIFT_MODULE__=1 Hello.swift 
Hello.swift:3:7: error: use of undeclared type 'dispatch_queue_t'
let x:dispatch_queue_t? = nil
      ^~~~~~~~~~~~~~~~
Hello.swift:5:10: error: use of unresolved identifier 'dispatch_group_create'
let g2 = dispatch_group_create()
         ^~~~~~~~~~~~~~~~~~~~~

```